src/components/Books.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import BookItem from './BookItem';
+import NewBook from './NewBook';
+
+function Books() {
+  return (
+    <div>
+      <BookItem />
```

Comment:
- Please render the `BookItem` in a list. Meaning, the `BookItem` should be wrapped in a list item `li`.